### ISSUE TITLE
Spell check only certain file types

### ISSUE THREAD

**Issue by GMNGeoffrey** [geany/geany-plugins#422]
I love this plugin for writing tex in geany, but find it a bit annoying when I'm coding and everything is underlined. It would be great if it could be configured to only spellcheck specific file types.

**Comment by eht16** [comment]
Basically SpellCheck should be clever enough for most filetypes to not check real code but instead only check words within comments and strings.

Still, the request sounds reasonable. Though I think I won't work on it any time soon but patches are welcome.

**Comment by GMNGeoffrey** [comment]
Yeah the "everything is underlined" was a bit hyperbolic :-P Enough of my comments and strings use things that aren't English words that it frequently hurts more than it helps.  For now I just toggle it off. I haven't explored the great world of geany plugin development, but perhaps I'll look into a patch at some point. Thanks for the reply!
